core/src/epicli/data/common/ansible/playbooks/roles/sshtunnel/files/tunnel.sh

```diff
@@ -0,0 +1,8 @@
+if [ "$1" = "--kill" ]; then
+  ps aux | 
+  grep -P "(/usr/bin/ssh|/usr/lib/autossh/autossh)\s.*$2" |
+  awk '{print $2}' |
+  xargs -r kill
+else
+  $(which autossh) "$@"
+fi
```

Comment:
```suggestion
#!/bin/bash

set -o errexit

if [ "$1" = "--kill" ]; then
  ps -C ssh,autossh -o pid,cmd |
  grep -E "@${2}($|[[:blank:]])" |
  awk '{print $1}' |
  xargs -r kill
else
  autossh "$@"
fi
```
Not tested as script